src/museums.ts

```diff
@@ -1,48 +1,91 @@
-import {
-  Curator,
-  Museum,
-  Origami,
-  utils,
-} from '@open-oracle-origami/origami-js-sdk'
+import { Curator, Museum, Origami } from '@open-oracle-origami/origami-js-sdk'
+import { ThirdwebSDK } from '@thirdweb-dev/sdk'
+import { TenetTestnet } from '@thirdweb-dev/chains'
 
 // TODO: this becomes it's own evm library or maybe it's an evm price aggregate library...
 // TODO: build threshold and deviation helpers
 // TODO: load from toml config
-const museumCollectionConfig = [
+const museumCollectionConfigs = [
   {
     id: 'tenet-testnet', // required
-    abi: [], // this can default to origami-solidity if not passed
-    seedPhrase: 'hello world', // required unless privateKey
-    privateKey: '0x0000000', // required unless seedPhrase
+    abi: fetch(
+      'https://raw.githubusercontent.com/open-oracle-origami/origami-solidity/main/abi/CollectionImplV1.json'
+    ).then(response => response.json()),
     deviation: 0,
```

Comment:
As a side note, these could also be configured individually in the collection, the museum top level is the default.